latest/setup/Dockerfile

```diff
@@ -4,4 +4,4 @@ RUN apk add -U curl bash sed jq
 COPY ./supply-files /supply-files
 COPY ./setup-zoomdata.sh /setup-zoomdata.sh
 
-CMD /setup-zoomdata.sh
+CMD ["./setup-zoomdata.sh"]
```

Comment:
@anjaneyuluuppu Adding `.` to CMD script call (./setup-zoomdata.sh) may drive to failures in case of changed working directory somewhere above in the Dockerfile. I've intentionally put it in the root folder (/) to call explicitly (/setup-zoomdata.sh). Also putting the command in `[]` brackets (forming the array) is useful only when the command contains some arguments which isn't our case.